packages/apps/plugins/plugin-graph/src/graph/types.ts

```diff
@@ -99,7 +99,7 @@ export namespace Graph {
      * @example ['unknown node label, { ns: 'example-plugin' }]
      */
     // TODO(thure): `Parameters<TFunction>` causes typechecking issues because `TFunction` has so many signatures.
-    label: string | [string, { ns: string; count?: number }];
+    label?: string | [string, { ns: string; count?: number }];
```

Comment:
Why optional?

Reply:
sorry typo as I was messing around understanding this